l3kernel/l3token.dtx

```diff
@@ -649,6 +649,43 @@
 %   for an undefined token.
 % \end{function}
 %
+% \begin{function}[EXP, pTF, added = 2025-05-12]{\token_if_control_symbol:N}
+%   \begin{syntax}
+%     \cs{token_if_control_symbol_p:N} \meta{token}
+%     \cs{token_if_control_symbol:NTF} \meta{token} \Arg{true code} \Arg{false code}
+%   \end{syntax}
+%   Tests whether the \meta{token} consists of the escape character followed by
+%   one non-letter character (called a \enquote{control symbol}).
```

Comment:
Strictly speaking it doesn't test this. It tests whether the token is a possibly undefined macro which's name consists of one non-letter. It still works (I hope!) on `\escapechar=-1`...